plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/rows/functions.tsx

```diff
@@ -126,6 +128,44 @@ function getStatusBadge( subscription: Subscription ): StatusBadge | false {
 			),
 		};
 	}
+	if ( subscription.expiring && ! subscription.autorenew ) {
+		return {
+			text: __( 'Expires soon', 'woocommerce' ),
+			level: StatusLevel.Error,
+			explanation: createInterpolateElement(
+				__(
+					'To receive updates and support, please <renew>renew</renew> this subscription before it expires or use a subscription from another account by <sharing>sharing</sharing> or <transferring>transferring</transferring>.',
+					'woocommerce'
+				),
+				{
+					renew: (
+						<a
+							href={ enableAutorenewalUrl( subscription ) }
+							rel="nofollow noopener noreferrer"
+						>
+							renew
+						</a>
+					),
+					sharing: (
+						<a
+							href={ MARKETPLACE_COLLABORATION_PATH }
```

Comment:
Shouldn't this be `MARKETPLACE_SHARING_PATH`?

Reply:
Nice catch! I copied it from another place and will fix it there now too 🙌

Reply:
Done in ehttps://github.com/woocommerce/woocommerce/pull/49985/commits/692c55300ca5e06c62074c5152d7a7045d1a8cb7